app/adapters/application.js

```diff
@@ -1,8 +1,14 @@
 import DS from 'ember-data';
+import Ember from 'ember';
 import config from '../config/environment';
 
 export default DS.RESTAdapter.extend({
-  host:'https://api.github.com',
```

Comment:
Can we store this in `config/environment.js`?  That allows us to have a different value in the test environment from development/production, and should allow you to remove the `host` computed.